groups:
  - Commands:
    - Autonomy:
        condition_name: Autonomy Commanded
    - Execute Mission:
        condition_name: Execute Mission Commanded